<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCS</class>
 <widget class="QDialog" name="GCS">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>663</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ground Control monitor</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="video_frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>UI/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="Drone_state_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>941</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Drone state</string>
   </property>
   <widget class="QLabel" name="Location_lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="Altitude_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>Altitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="Location">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="Altitude">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="Attitude_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>Atttitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="Attitude">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="VehiclaMode_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>VehiclaMode</string>
    </property>
   </widget>
   <widget class="QLabel" name="VehiclaMode">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="Battery_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
   </widget>
   <widget class="QLabel" name="Battery">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>319</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Drone connection</string>
   </property>
   <widget class="QLineEdit" name="Port">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8888</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="IP_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192.168.50.25</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="title">
     <string>Anomaly</string>
    </property>
    <widget class="QLabel" name="Anomaly">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string> -</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="open_cam_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>open camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_folder_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>open folder</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
